vast/vastelement/vast.go

```diff
@@ -44,9 +44,12 @@ func (v Vast) FindFirstInlineCompanionAdsCreative() *CompanionAds {
 	return nil
 }
 
+// MediaFileChecker is a custom filter, input is MediaFile and the output is an error.
```

Comment:
```suggestion
// MediaFileChecker is a custom filter, input is MediaFile and the output is an error. If validation failed it will return an error otherwise it will return nil.
```